src/app/api/voice/route.ts

```
@@ -66,16 +94,19 @@ export async function POST(request: NextRequest) {
         const result = await realtimeAgent.processVoiceInput(audioBuffer);
         
         // Convert audio response back to base64
-        const audioResponseBase64 = Buffer.from(result.audioResponse).toString('base64');
+        const audioResponseBase64 = Buffer.from(new Uint8Array(result.audioResponse)).toString('base64');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)